iree/compiler/Dialect/Flow/Transforms/LegalizeInputTypes.cpp

```diff
@@ -83,6 +84,13 @@ LogicalResult convertOperation(Operation *oldOp,
                                FlowTypeConverter &typeConverter,
                                BlockAndValueMapping &mapping,
                                OpBuilder &builder) {
+  if (llvm::isa<mlir::linalg::LinalgOp>(oldOp)) {
```

Comment:
Wondering why the Linalg op will need to do something different from the logic below. If the op has valid types as assumed here it should just work?